src/static/_redirects

```diff
@@ -12,33 +9,4 @@ https://eslint.netlify.com/* https://eslint.org/:splat 301!
 /chat                               https://discord.gg/8szcydm 302!
 
 # Internal Redirects
-/docs                               https://eslint.org/ 301!
-/docs/user-guide/rules              https://eslint.org/docs/rules 301!
-/docs/rules/id-blacklist            https://eslint.org/docs/rules/id-denylist 301!
-
-/docs/0.24.1/command-line-interface https://eslint.org/docs/user-guide/command-line-interface 301!
-/docs/0.24.1/configuring            https://eslint.org/docs/user-guide/configuring 301!
-/docs/0.24.1/integrations           https://eslint.org/docs/integrations 301!
-/docs/0.24.1/user-guide/rules       https://eslint.org/docs/rules 301!
-
-/docs/1.0.0/command-line-interface  https://eslint.org/docs/user-guide/command-line-interface 301!
-/docs/1.0.0/configuring             https://eslint.org/docs/user-guide/configuring 301!
-/docs/1.0.0/integrations            https://eslint.org/docs/integrations 301!
-/docs/1.0.0/user-guide/rules        https://eslint.org/docs/rules 301!
-
-/docs/1.10.3/command-line-interface https://eslint.org/docs/user-guide/command-line-interface 301!
-/docs/1.10.3/configuring            https://eslint.org/docs/user-guide/configuring 301!
-/docs/1.10.3/integrations           https://eslint.org/docs/integrations 301!
-/docs/1.10.3/user-guide/rules       https://eslint.org/docs/rules 301!
-
-/docs/2.0.0/command-line-interface  https://eslint.org/docs/user-guide/command-line-interface 301!
-/docs/2.0.0/configuring             https://eslint.org/docs/user-guide/configuring 301!
-/docs/2.0.0/integrations            https://eslint.org/docs/integrations 301!
-/docs/2.0.0/user-guide/rules        https://eslint.org/docs/rules 301!
-
-/docs/2.13.1/command-line-interface https://eslint.org/docs/user-guide/command-line-interface 301!
-/docs/2.13.1/configuring            https://eslint.org/docs/user-guide/configuring 301!
-/docs/2.13.1/integrations           https://eslint.org/docs/integrations 301!
-/docs/2.13.1/user-guide/rules       https://eslint.org/docs/rules 301!
-
-/docs/3.0.0/user-guide/rules        https://eslint.org/docs/rules 301!
\ No newline at end of file
+/docs/*                               https://eslint.org/ 301!
```

Comment:
```suggestion
/docs/*                               https://eslint.org/docs/:splat 302!
```

We need to cover all docs URLs and definitely don’t want this to be permanent.

Reply:
Done